cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+If the expression to the right of a comma operator starts at an earlier column than the expression to the left, then
+this suspicious indentation possibly indicates a logic error, caused by a typo that may escape visual inspection.
+</p>
+<warning>
+This query has medium precision because CodeQL currently does not distinguish between tabs and spaces in whitespace.
+Alerts may therefore flag code that appears readable for one value of tab size but not another.
+</warning>
+</overview>
+
+<recommendation>
+<p>
+Use standard indentation around the comma operator: begin the right-hand-side operand at the same level of
+indentation (column number) as the left-hand-side operand.
+</p>
+<p>
+When it comes to whitespace, either do not mix tabs and spaces, or mix them consistently.
+</p>
+</recommendation>
+
+<example>
+<sample src="CommaBeforeMisleadingIndentation.cpp" />
+</example>
+
```

Comment:
Are there any articles or guidelines that we could refer to? I had a quick look but couldn't find anything.

Reply:
Not very targeted, but I added references to Wikipedia.